src/services/ApiService/History/types.ts

```diff
@@ -0,0 +1,27 @@
+import { ITxStatus, TAddress, ITxHash, ITxType } from '@types';
+
+export interface ITxHistoryApiResponse {
+  readonly blockNumber?: string; // Hex
+  readonly data: string;
+  readonly erc20Transfers?: ITxHistoryERC20Transfer[];
+  readonly from: TAddress;
+  readonly gasLimit: string; // Hex
+  readonly gasPrice: string; // Hex
+  readonly gasUsed?: string; // Hex
+  readonly hash: ITxHash;
+  readonly nonce: string; // Hex
+  readonly recipientAddress: TAddress;
+  readonly status: ITxStatus.PENDING | ITxStatus.SUCCESS | ITxStatus.FAILED | ITxStatus.UNKNOWN;
+  readonly timestamp?: number;
+  readonly to: TAddress;
+  readonly value: string; // Hex
+
+  readonly txType: ITxType;
+}
```

Comment:
In my last pr i added these which i think would be good to stick here: https://github.com/MyCryptoHQ/MyCrypto/blob/master/src/types/transactionFlow.ts#L17-L23